andela/ah-frontend-zeus
# #162163122 Setup an empty React project

## What does this PR do?
Sets up an empty React project
### Description of the task to be completed?
- Add .gitignore, README.md, and LICENSE.txt
- Initialize repository as an npm project (adds package.json)
- Add dependencies for Webpack, Babel and React to package.json
- Create .babelrc to configure Babel
- Add Webpack configuration to webpack.config.js
- Create basic application files (index.html, index.js, App.js)
- Configure hot reloading in App.js
- Setup ESLint using Airbnb rules
### How should this be manually tested?
- Install all dependencies using **yarn**
- Run the command `yarn start` to execute the development server
- Open `localhost:3000` in a browser to verify that the app is running
- To lint, use the command: `yarn run lint`
### Screenshot
<img width="453" alt="app screenshot" src="https://user-images.githubusercontent.com/34346070/50838353-e329ab80-136e-11e9-8ea2-2b5b8ed0eb1c.png">

### What are the relevant pivotal tracker stories?
[#162163122](https://www.pivotaltracker.com/story/show/162163122)
